spring-cloud-previews/google-cloud-apigee-connect-spring-starter/src/main/java/com/google/cloud/apigeeconnect/v1/spring/TetherSpringAutoConfiguration.java

```diff
@@ -92,9 +92,6 @@ protected TetherSpringAutoConfiguration(
   @Bean
   @ConditionalOnMissingBean(name = "defaultTetherTransportChannelProvider")
   public TransportChannelProvider defaultTetherTransportChannelProvider() {
-    if (this.clientProperties.getUseRest()) {
```

Comment:
This corresponds to `useRestOption` update in the generator: see https://github.com/googleapis/gapic-generator-java/pull/1343 for details.